src/leo-client-app/src/components/SatellitesOfInterest.tsx

```diff
@@ -0,0 +1,113 @@
+"use client";
+
+import { Card, CardContent, Grid, Stack } from "@mui/material";
+import Link from "next/link";
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import "../styles.css";
+import "./styles/satellitesOfInterest.css";
+import "./styles/component.css";
+import UserName from "./UserName";
+
+type Props = {
+  userId: string;
+};
+
+type SatelliteDetails = {
+  name: string;
+  noradId: string;
+};
+
+const SatellitesOfInterest = ({ userId }: Props) => {
+  const [satellites, setSatellites] = useState<SatelliteDetails[]>([]);
+
+  const fetchSatellites = async (satelliteId: string) => {
+    try {
+      const response = await axios.get(
+        "http://localhost:3001/satellite/getSatellite",
```

Comment:
Can you reference the backend using BACKEND_URL (from constants/api.ts) like `${BACKEND_URL}/satellite/getSatellite`, since this export will later be changed to reflect the backend url stored in the env